### DISCUSSION TITLE
Viewing changes preview in website

### DISCUSSION THREAD

**Discussion by wesleyhuck** [opengisch/QField#4896] [Q&A]
Hello, I’m new to using QField and I’m building a model to collect data from the field to improve the city’s water network. The problem is that the first time I tested the app on my smartphone, the changes I made appeared in the changes menu on the website with a preview on the map that helped me find where the changes were made. However, now they don’t appear anymore when I click on the change ID. Is this feature discontinued? If yes, there's a way to help me find the location on where the changes were made?

**Comment by w0pr** [comment]
> the changes I made appeared in the changes menu on the website with a preview on the map that helped me find where the changes were made. However, now they don’t appear anymore when I click on the change ID.

Are you talking about the qfield.cloud site and the changes ID circled in red?
![image](https://github.com/opengisch/QField/assets/8257055/b88ffa55-51b5-4888-b4a7-842b2a3b3fc9)

Try another browser. It seems that since at least a few weeks ago, opening this popup does not work anymore in Chrome (tried with current version120.0.6099.130), since the relative URL (from "href" property) is missing its base.

![image](https://github.com/opengisch/QField/assets/8257055/2d213a4c-fd6d-4491-a308-f0d83d4eb1e9)

```
jquery.min.js:2 Uncaught TypeError: Failed to construct 'URL': Invalid URL
    at new Reviewer (qfc.0e04ee1f1fa0.js:860:21)
    at qfc.0e04ee1f1fa0.js:1381:22
    at HTMLDocument.<anonymous> (qfc.0e04ee1f1fa0.js:1453:5)
    at e (jquery.min.js:2:30038)
    at t (jquery.min.js:2:30340)
```

Firefox works, though.

**Reply by wesleyhuck** [reply]
That’s exactly what I needed. I switched to Firefox and it works perfectly now. Thank you for your help!
